library/ssl_misc.h

```diff
@@ -2085,7 +2085,7 @@ static inline int mbedtls_ssl_sig_alg_is_supported(
 #define MBEDTLS_SSL_SIG_ALG( hash )
 #endif /* MBEDTLS_ECDSA_C && MBEDTLS_RSA_C */
 #endif /* MBEDTLS_SSL_PROTO_TLS1_2 && MBEDTLS_KEY_EXCHANGE_WITH_CERT_ENABLED */
-#if defined(MBEDTLS_USE_PSA_CRYPTO)
+#if defined(MBEDTLS_PSA_CRYPTO_C)
```

Comment:
I think we need to use the same guard on line 30 around the inclusion of PSA headers - the CI currently has errors in configurations where `MBEDTLS_USE_PSA_CRYPTO` is disabled.